sessions/libsessions_test.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func TestLibSessionSGetSetCGRID(t *testing.T) {
-	sEv := engine.NewSafEvent(map[string]interface{}{
+	sEv := engine.NewMapEvent(map[string]interface{}{
```

Comment:
Please use in all tests engine.MapEvent(map...) instead of New since that one will only apply another if.